usecases/schema/executor.go

```diff
@@ -91,7 +90,7 @@ func (e *executor) UpdateClass(req api.UpdateClassRequest) error {
 		}
 	} else {
 		if err := e.migrator.UpdateVectorIndexConfig(ctx,
-			className, req.Class.VectorIndexConfig.(schemaConfig.VectorIndexConfig)); err != nil {
+			className, asVectorIndexConfig(req.Class)); err != nil {
```

Comment:
question(non-blocker): did this crash weaviate ?

Reply:
yup